### ISSUE TITLE
Implement ProjectMembers View

### ISSUE THREAD

**Issue by robyngit** [NCEAS/metacatui#821]
The Project Members View will list the name, organization, contact information, as well as the role in the project for each of the project's members.

This is a subtask of NCEAS/project-papers#35

**Comment by jkibele** [comment]
I guess I should've checked before I made #825

**Comment by jkibele** [comment]
Hey @robyngit , I had a merge conflict when I tried to push my version of ProjectMembersView. I chose my changes (for the reasons in the commit - 76b605fc9630ff023078dea5131c54f016d1348b). It seemed to go okay, but now my working directory has a bunch of changes that I didn't make:

![image](https://user-images.githubusercontent.com/4845845/47946130-a9e34400-dec4-11e8-93b1-36e50699ff0e.png)

Are those files that you changed? Did my ham-fisted merge take them out of the commit or something?
